tests/Aspire.Hosting.Tests/Dashboard/DashboardResourceTests.cs

```diff
@@ -0,0 +1,191 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Dcp;
+using Aspire.Hosting.Tests.Utils;
+using Aspire.Hosting.Utils;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Aspire.Hosting.Tests.Dashboard;
+
+public class DashboardResourceTests
```

Comment:
Consider adding tests for scenarios where dashboard isn't added, e.g. manifest mode and dashboard disabled flag.

Reply:
Also a test for when there is a `aspire-dashboard` project already added to builder.

Reply:
Done